try {
    console.log(a.b);
} catch (error) {
    console.log(error.message);
}